mrc/localization/position/probability.py

```diff
@@ -4,21 +4,65 @@
 
 
 class AngularProbability:
+    """
+    Class responsible for calculation probability of robot being in given angle
+
+    Intended for internal usage.
+    """
     def __init__(self, stdev, mean, coef):
+        """
+        Parameters
+        ----------
+        sigma : float
+            Gaussian function parameter
+        mu : float
+            Gaussian function parameter
+        coef : float
+            Range expansion coefficient
+        """
         self.mean = mean
         self.stdev = stdev
         self.coef = coef
 
     def __call__(self, rotation):
+        """
+        Calculate probability of robot being in given position
+
+        Parameters
+        ----------
+        rotation : float
+            Angle at which we see other robot.
+
+        Returns
+        -------
+        float
+            Probability in given rotation_angle
+        """
         d = math.pi - self.mean
         p1 = gauss_distribution(rotation, self.mean, self.stdev)
         p2 = gauss_distribution(rotation, -math.pi - d, self.stdev)
         return max(p1, p2)
 
     def reset(self):
+        """
+        Reset coefficient to original value
+        """
         self.stdev /= self.coef
```

Comment:
I'm not sure if that's correct but lets let it be for now

Reply:
Okay